src/components/Reports/GoalCalculator/GoalCalculatorTestWrapper.tsx

```diff
@@ -45,9 +58,13 @@ export const GoalCalculatorTestWrapper: React.FC<GoalCalculatorTestWrapper> = ({
       }}
     >
       <SnackbarProvider>
-        <GqlMockedProvider<{ GoalCalculation: GoalCalculationQuery }>
+        <GqlMockedProvider<{
+          GoalCalculation: GoalCalculationQuery;
+          UpdateAccountPreferences: UpdateAccountPreferencesMutation;
+        }>
           mocks={{
             GoalCalculation: goalCalculationMock,
+            UpdateAccountPreferences: updateAccountPreferencesMock,
```

Comment:
It doesn't look like your test cares what the update mutation response is, so we don't need to mock the response. You can revert your changes to this file.